docs/sources/panels/visualizations/table-panel.md

```diff
@@ -27,6 +27,43 @@ Table visualizations allow you to apply:
 
 - **Show header -** Show or hide column names imported from your data source..
 
+### Custom field options
```

Comment:
This content does not belong here. Please remove.

Reply:
It does. These are table visualization settings. I think they should be together with the other table display settings.

Reply:
I just proposed https://github.com/grafana/grafana/pull/27579 to help make this more clear in the UI :)